src/Middleware/Csrf.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Yiisoft\Yii\Web\Middleware;
+
+use Psr\Http\Message\ResponseFactoryInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\MiddlewareInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Yiisoft\Security\Random;
+use Yiisoft\Security\TokenMasker;
+use Yiisoft\Yii\Web\Cookie;
+use Yiisoft\Yii\Web\Session\SessionInterface;
+
+final class Csrf implements MiddlewareInterface
+{
+    public const CSRF_HEADER = 'X-CSRF-Token';
+
+    private $name = '_csrf';
+    private $cookieParams = ['httpOnly' => true];
```

Comment:
Should there be a way to change it? If yes, setter is needed. If no, it could be a private constant.

Reply:
What do you think, should be or not?

Reply:
This particular parameter I think should not be changed ever.